contracts/license.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity ^0.5.0;
+
+import "./ownable.sol";
+
+/**
+* @title License
+* @dev License contract for buying a license
+*/
+contract License is Ownable {
```

Comment:
Is there some kind of limitation on the number of licenses? Should there be a way for a licenseOwner to revoke their license and get the fee back?

Reply:
This is only about buying a license